keep/api/models/db/migrations/versions/2024-11-08-20-58_895fe80117aa.py

```diff
@@ -0,0 +1,41 @@
+"""Add timestamp and provider_type to alertraw
+
+Revision ID: 895fe80117aa
+Revises: ef0b5b0df41c
+Create Date: 2024-11-08 20:58:40.201477
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "895fe80117aa"
+down_revision = "ef0b5b0df41c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    with op.batch_alter_table("alertraw", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column(
+                "timestamp",
+                sa.DateTime(),
+                nullable=False,
+                server_default=sa.text("CURRENT_TIMESTAMP"),
+            )
+        )
+        batch_op.add_column(sa.Column("provider_type", sa.String(), nullable=True))
```

Comment:
in MySQL it fails over:
`VARCHAR requires a length on dialect mysql`
```suggestion
        batch_op.add_column(sa.Column("provider_type", sa.String(255), nullable=True))
```